docs/configuration.rst

```diff
@@ -8,6 +8,11 @@ Requirements
 * access to Pulp 2 database
 * [recommended] all Pulp 3 supported checksum types are allowed
 
+.. note::
+
+    The migration workload is very I/O intensive. It's highly recommended that the volume
+    backing your Postgresql database be on a SSD or other low latency, high IOPs storage volume.
```

Comment:
Should it be `IOPS`? I'm not sure but I think I recall seeing it in uppercase only.

Reply:
It should be `IOPS` since second is part of the acronym. :+1: